.coveragerc

```diff
@@ -11,3 +11,13 @@ omit =
     # TODO(#13935): Remove python_utils from the list once the codebase is
     # run under Python 3.
     core/python_utils.py
+
+# exclude_lines is used to exclude a particular clause.
+# Any line with a comment of “pragma: no cover” is excluded.
+# If that line introduces a clause, for example, an if clause, function or class,
+# then the entire clause is also excluded. Similarly, @overload is used to exclude 
+# a particular function's multiple signatures, which are skipped during runtime.
```

Comment:
```suggestion
# The 'exclude_lines' is used to skip a particular clause in coverage.
# We exclude any line with a comment of “pragma: no cover” as this is the default option
# provided by the coverage library.
# We also exclude @overload, because it is used for defining multiple MyPy definitions 
# of a function and cannot be tested by Python tests.
```